packages/backend-app-api/src/config/config.ts

```diff
@@ -70,6 +71,7 @@ export async function createConfigSecretEnumerator(options: {
 export async function loadBackendConfig(options: {
   remote?: LoadConfigOptionsRemote;
   argv: string[];
+  config?: JsonObject;
```

Comment:
Thinking it's slightly more flexible if we make this `additionalConfig?: AppConfig` instead. It's slightly lower level, but this is a pretty niche customzation point anyway